Rovers/Rovers.tex

```diff
@@ -0,0 +1,405 @@
+    Long range communications & Ability to communicate with the rover at lengths of up to one kilometer allows for URC readiness. \\
+    \hline
+    \end{tabularx}
+    }
+\label{tab:mvp-two}
+\end{table}
+
+\section{Secondary Objectives}
+\label{sec:secondary-obj}
+The study will also investigate the University Rover Competition (URC) as hosted by the Mars Society as a potential long term goal. 
+The team will accomplish this by evaluating how many of the MVP improvements are accomplished. 
+If none of these improvements are able to be implemented then the team is not near URC ready.
+The competition is held annually and features four very intense competitions. 
+Such a rover would need to be fully self navigating, perform scientific analysis on soil samples, and having a robotic arm capable of fine motor control. 
+Such a project would be among the most ambitious SPEX has ever attempted. 
+These features line up with the improvements section of the MVP.
+The skills learned from this project should be applicable to such a future project. 
+This secondary objective is directly related to the long-term vision of the rover project. 
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+
+\subsection{Space Exploration}
+\label{subsec:spacex}
+Rovers are a huge part of space exploration. 
+There are eight active missions on Mars. 
+Four of those include rovers. 
+To ignore this fact would be to ignore a large part of space exploration.
+Currently RIT SPEX is not involved with rovers.
+It would be beneficial to our members to get some experience in this area.
+
+\subsection{Promotions and Funding}
+\label{subsec:promos}
+A rover would be a popular demonstration at ImagineRIT.
+The rover would be rather large and would attract many eyes. 
+We could even demo it outside if there is sufficient space. 
+A rover would be very easy to get video and photos of for SPEX promotions. 
+Having a rover is also another opportunity for SPEX to fundraise. 
+There is plenty of space to place company logos on the body of the rover. 
+It would also allow for SPEX to reach out to robotics companies.
+
+\subsection{Computer Science}
+\label{subsec:cs}
+\subsubsection{Member Retention and Expansion}
+\label{subsubsec:cs}
+The potential self-driving component would be the heaviest computer science project SPEX would have attempted, this would help with retention of CS and SE majors. 
+There are many features included in the MVP that require significant computer science skills as well.
+This includes navigation, power management, communications and more.
+That will give computing-related majors another option for SPEX involvement, also helping retention.
+
+\subsubsection{Artificial Intelligence}
+\label{subsubsec:ai}
+Some of the potential self-driving features will likely be implemented with artificial intelligence. In particular the fully autonomous navigation would require this.
```

Comment:
What specific features would be implemented using AI? What data sets and open-source technologies would be used? How does 'AI' make the rover move autonomously?

Reply:
OpenCV, it detects and loggs tennis balls